lib/active_shipping/carriers/fedex.rb

```diff
@@ -153,10 +161,129 @@ def find_tracking_info(tracking_number, options = {})
       parse_tracking_response(xml, options)
     end
 
+
+    # Get Shipping labels
+    # Caveats:
+    #  - Only supports singlular packages
```

Comment:
Yes UPS definitely does support this. They have the concept of a Shipment that contains multiple Packages. I think that Fedex supports it as well, but this is one of many features I didn't have time to implement.
We could replace the comment with an error that says the functionality is not implemented.